### ISSUE TITLE
Exception calling "Substring" with "1" argument(s) when adding migration

### ISSUE THREAD

**Issue by gecko-8** [domaindrivendev/Swashbuckle.AspNetCore#394]
I'm using Swashbuckle.AspNetCore in a project with EF code first migrations. Whenever I add a new migration I get the following error (the migration is correctly created):

```
PM> add-migration AddedTest -Context AxiomDbContext
dbug: Swagger Documentation[0]
add-migration : Exception calling "Substring" with "1" argument(s): "StartIndex cannot be less than zero.
Parameter name: startIndex"
At line:1 char:1
+ add-migration AddedTest -Context AxiomDbContext
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Add-Migration], MethodInvocationException
    + FullyQualifiedErrorId : ArgumentOutOfRangeException,Add-Migration

```
I originally got this error with Swashbuckle.Swagger and at the time found a post somewhere that indicated it was an issue with the Swashbuckle library and that it was fixed in the AspNetCore library. I would tend to agree based on the "Swagger documentation" line in the error. Unfortunately I can no longer find that post but I'm still getting the error after upgrading to Swashbuckle.AspNetCore.

I'm using the following versions:
Swashbuckle.AspNetCore: v1.0.0
Microsoft.AspNetCore: v1.1.1
NETStandard.Library: v1.6.1

Is this a known issue? Or is there some further information I can provide?

**Comment by domaindrivendev** [comment]
I have no idea how the EF CLI works and why Swashbuckle would be impacting it. So, I don't think I'll be able to offer much help. Have you confirmed that this error doesn't occur when Swashbuckle is uninstalled and/or disabled?

Also, given the time since the original issue - you may want to pull down the latest version (v2.5.0 as of writing) and see if it's still persisting for you

**Comment by gecko-8** [comment]
@domaindrivendev Yes, I confirmed it by commenting out the couple Swashbuckle lines in the Startup.cs file. After I did that the migrations would work fine. That's actually my workaround for anytime I needed to work with migrations. 

That project is currently on hold so I'm not sure when I'll get a chance to try the latest version. I did try several new versions after reporting the issue (several months worth).
